### ISSUE TITLE
lxqt keyboard layout issue

### ISSUE THREAD

**Issue by cas1no** [NixOS/nixpkgs#150390]
### Describe the bug
When using lxqt, I cannot add keyboard layouts to `Keyboard and Mouse Settings`.

### Steps To Reproduce
1. Open `Keyboard and Mouse Settings`.
2. Press `Add` button, and you'll see an empty list.

### Expected behavior
After pressing `Add` button there should be a list of languages.

### Additional context
As I can see, this happens, because the path to the `xkeyboard-config` is hard-coded in the lxqt itself.
https://github.com/lxqt/lxqt-config/blob/d11e93cea008432161907c96b749e4a28b2f6e58/lxqt-config-input/keyboardlayoutconfig.h#L26

Probably, this file should be patched, then the issue will be resolved.

**Comment by romildo** [comment]
@cas1no
Please, test the fix in https://github.com/NixOS/nixpkgs/pull/152345

**Comment by cas1no** [comment]
> @cas1no Please, test the fix in #152345

Sorry, I'm a bit newbie with the nixos, I don't know how to test it 😅
Maybe you can tell me how to do it? Then I'll do it ASAP

**Comment by romildo** [comment]
Usually I test changes to desktop environments with a virtual machine.

1. First I need a local copy of the nixpkgs repository.
   ```console
   $ git clone https://github.com/NixOS/nixpkgs.git
   $ cd nixpkgs
   ```
2. Then I get a copy of the PR branch
   ```console
   $ git checkout -b romildo-upd.lxqt-config master
   $ git pull https://github.com/romildo/nixpkgs.git upd.lxqt-config
   ```
4. I also prepare a configuration file for a basic machine: `configuration-lxqt.nix`
   ```nix
   { config, pkgs, ... }:

   {
     fileSystems."/" = { device = "/dev/sda1"; fsType = "ext4"; };
       
     boot.loader.systemd-boot.enable = true;
     boot.loader.efi.canTouchEfiVariables = true;

     environment.systemPackages = with pkgs; [
       vim
     ];

     services.xserver.enable = true;
     services.xserver.displayManager.lightdm.enable = true;
     services.xserver.desktopManager.lxqt.enable = true;

     users.extraUsers.guest = {
       isNormalUser = true;
       uid = 1100;
       initialPassword = "g";
       extraGroups = [ "wheel" ];
     };
   }
   ```
4. Then I build the system as a virtual machine:
   ```console
   $ nixos-rebuild build-vm --option sandbox true -I nixpkgs=. -I nixos-config=configuration-lxqt.nix
   ```
5. And finally I run the virtual machine:
   ```console
   $ QEMU_OPTS="-m 1G,slots=3,maxmem=4G" result/bin/run-nixos-vm
   ```
